addons/mail/static/src/core/common/thread.js

```diff
@@ -181,6 +183,14 @@ export class Thread extends Component {
         return this.state.showJumpPresent ? PRESENT_THRESHOLD - 200 : PRESENT_THRESHOLD;
     }
 
+    get preferenceButtonText() {
+        const [, before, inside, after] =
+            _t(
+                "<button>Change your preferences</button> to receive new notifications in your Inbox."
+            ).match(/(.*)<button>(.*)<\/button>(.*)/) ?? [];
+        return { before, inside, after };
+    }
```

Comment:
This looks like a draft to generalize such strings for translation. Ok to keep it as-is for now, this is a rough idea of how a translation utils should help us handle such UI text elements.